lib/builder.dart

```diff
@@ -1,7 +1,3 @@
-// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
```

Comment:
This does need to be here as well, but should reference `Google Inc` instead of the Dart project authors:

```dart
// Copyright (c) 2015, Google Inc. Please see the AUTHORS file for details.
// All rights reserved. Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.
```